Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1551,6 +1557,22 @@ def get_tags_for_generic_context(tags: Optional[list]):
     return results
 
 
+def get_tags_for_tags_and_malware_family_fields(tags: Optional[list], is_malware_family=False):
+    """get specific tags for the tgas and malware_family fields"""
+    if not tags:
+        return None
```

Comment:
Why `None` and not `[]`?

Reply:
In any case in common classes it checks if it is None then it does not output  the field to the context data